lib/StaticNarrative/exporter/processor_util.py

```diff
@@ -66,45 +63,41 @@ def build_report_view_data(
     """
     if not result:
         return {}
-    if not isinstance(result, list):
-        result = [result]
+    if isinstance(result, list):
+        result = result[0]  # what about the rest of the list?!
     if (
-        not result[0]
-        or not isinstance(result[0], dict)
-        or not result[0].get("report_name")
-        or not result[0].get("report_ref")
+        not result
+        or not isinstance(result, dict)
+        or not result.get("report_name")
+        or not result.get("report_ref")
     ):
         return {}
-    report_ref = result[0]["report_ref"]
```

Comment:
didn't seem to be much point in having `result` as a list as it was only ever the first item that was accessed